documentation/modules/upgrading/proc-upgrade-brokers-newer-kafka.adoc

```diff
@@ -6,19 +6,19 @@
 
 = Upgrading Kafka brokers and client applications
 
-This procedure describes how to upgrade a Strimzi Kafka cluster to a higher version of Kafka.
+This procedure describes how to upgrade a Strimzi Kafka cluster to the latest supported Kafka version.
 
 .Prerequisites
 
 For the `Kafka` resource to be upgraded, check:
 
 * The Cluster Operator, which supports both versions of Kafka, is up and running.
-* The `Kafka.spec.kafka.config` does _not_ contain options that are not supported in the version of Kafka that you are upgrading to.
-* Whether the `log.message.format.version` and `inter.broker.protocol.version` for the current Kafka version need to be updated for the new version.
+* The `Kafka.spec.kafka.config` does _not_ contain options that are not supported in the new Kafka version.
+* Whether the `log.message.format.version` and `inter.broker.protocol.version` for the current Kafka version need to be updated for the new Kafka version.
```

Comment:
I wonder whether we're phrasing this right. It's not like these things necessarily _need_ to be changed. Rather it's up to the user to manage them and thus understand what they do and so decide when it's right for them to change them.

Reply:
I'm certainly not keen on the second bullet. How would the user find out which options are _not_ supported in the new Kafka version? By checking the Kafka release notes? Are we just covering our backs or is this an actual concern, I wonder.

As for the third bullet, I can rephrase the sentence and perhaps move it up to the introduction.

Reply:
Line 17 is now part of the procedure introduction:

> Compared to your current Kafka version, the new version might support a higher _log message format version_ or _inter-broker protocol version_, or both. Follow the steps to upgrade these versions, if required. For more information, see Kafka versions.